samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Steps/Pages/GoogleAuthenticator.cs

```diff
@@ -0,0 +1,76 @@
+﻿using embedded_auth_with_sdk.E2ETests.PageObjectModels;
+using TechTalk.SpecFlow;
+
+namespace embedded_auth_with_sdk.E2ETests.Steps.Pages
+{
+    [Binding]
+    public class GoogleAuthenticator : BaseTestSteps

```

Comment:
Should we add a suffix to this class to clarify this is the page/steps?

Reply:
@laura-rodriguez I would remove this suffix for all the other classes in the namespace instead.